api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1131,35 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
```

Comment:
As we will not have any other types of subscriptions I don't see need to make the name more specific.